### PR TITLE
PLASTIC parse_txt routine file

### PR DESCRIPTION
**Pull request by ankitkmr**
This function is intended to read in data from PLASTIC data file stored on hard disk and return the header as a list and data as pandas dataframe. I'll implement the class structure around it soon.

### CODE REVIEW

File: sunpy/lightcurve/sources/plastic.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    elif type_of_data == '10min':
+        for i in range(len(data)):
+            data['col5'][i] = datetime.strptime(str(data['col5'][i]), "%Y-%m-%d/%H:%M:%S")
+        
+        data.remove_columns(['col1','col2','col3','col4'])
+        header = ['Datetime'] + header[5:]
+
+    elif type_of_data == '1hr':
+        for i in range(len(data)):
+            data['col4'][i] = datetime.strptime(str(data['col4'][i]), "%Y-%m-%d/%H:%M:%S")
```
Comment (dpshelio) [inline sunpy/lightcurve/sources/plastic.py]: if the date format is the same everywhere, I would suggest to write it into a variable and use it in the other places

File: sunpy/lightcurve/sources/plastic.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    data_modify = []
+    data_modify_other = []
+
+    #Converting separate datetime element into a single datetime.datetime column
+    if type_of_data == '1min':
+        for i in range(len(data)):
+            data_modify = data_modify + [(datetime.strptime(str(data['col7'][i]) + str(data['col6'][i])[1:], "%Y-%m-%d/%H:%M:%S.%f") )]
+            data_modify_other = data_modify_other + [datetime.strptime(str(data['col8'][i]), "%Y-%m-%d/%H:%M:%S") ]
+        
+        data.remove_columns(['col1','col2','col3','col4','col5','col6','col7','col8'])
```
Comment (dpshelio) [inline sunpy/lightcurve/sources/plastic.py]:
```python
data.remove_columns(['col{}'.format(i) for i in range(1,9) ])
```

File: sunpy/lightcurve/sources/plastic.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+        f.readline()
+        header = f.readline().strip('\n').split('\t')
+
+    if header[4] == 'sec':
+        type_of_data = '1min'
+    elif header[4] == 'date and time':
+        type_of_data = '10min'
+    elif header[4] == 'Np [1/cc]':
+        type_of_data = '1hr'
+    else:
```
Comment (dpshelio) [inline sunpy/lightcurve/sources/plastic.py]:
```python
types = {'sec':'1min', 'date and time': '10min', 'Np [1/cc]': '1hr'}
```
and then 
```python
type_of_data = types.get(header[4])
if type_of_data is None:
    raise ValueError('{var} is not on of the possibilities in the header it should be one of {poss}'.format(var=header[4], poss= types.keys()))
```
or 
```python
try:
    type_of_data = types[header[4]]
except KeyError:
    print('the header is not blablabala')
    raise
```

File: sunpy/lightcurve/sources/plastic.py [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+            data_modify_other = data_modify_other + [datetime.strptime(str(data['col8'][i]), "%Y-%m-%d/%H:%M:%S") ]
+        
+        data.remove_columns(['col1','col2','col3','col4','col5','col6','col7','col8'])
+        data.add_column(Column(data = data_modify, name='col_1'),0)
+        data.add_column(Column(data = data_modify_other, name='col_2'),1)
+        header = ['Datetime'] + header[7:]
+        
+    elif type_of_data == '10min':
+        for i in range(len(data)):
+            data['col5'][i] = datetime.strptime(str(data['col5'][i]), "%Y-%m-%d/%H:%M:%S")
```
Comment (dpshelio) [inline sunpy/lightcurve/sources/plastic.py]: could not you use:
```python
data['col5'] = [datetime.strptime(str(var), "%Y-%m-%d/%H:%M:%S") for var in data['col5']]
```

**Comment by ehsteve** [comment]
This should be in a STEREO module not its own module.

**Comment by ehsteve** [comment]
PRs require a description.

File: sunpy/lightcurve/sources/plastic.py (hunk 1 above)
Comment (ankitkmr) [inline sunpy/lightcurve/sources/plastic.py]: Made all the changes :)

**Pull request closed without merging**